packages/input-checkbox/AUTHORS

```diff
@@ -1,4 +1,4 @@
-Crave Food Services:
+Crave Food Systems:
```

Comment:
cool, I've filed https://github.com/CraveFood/farmblocks/issues/212 to remember to check other places